src/relay_sdk/events.py

```diff
@@ -13,18 +13,27 @@ class Events:
     def __init__(self, client: Session) -> None:
         self._client = client
 
-    def emit(self, data: Mapping[str, Any]) -> None:
-        """Create and send an event to the service.
+    def emit(self, data: Mapping[str, Any], key: Any = None) -> None:
+        """Sends an event to the service.
 
         Use this from a Trigger handler to start the workflow
         associated with the trigger.
 
-        Accepts a Mapping of data fields that will bind
-        names of workflow parameters to values from the
-        request payload that came in to the Trigger handler"""
+        Accepts a Mapping of data fields from the request payload
+        that came in to the Trigger handler.
+
+        A key can be optionally provided to uniquely identify
+        the event."""
+
+        if key is not None:
```

Comment:
Even though Python lets you do this, you typically wouldn't want to overwrite a dict that comes from an external argument here for clarity. The (IMO) most Pythonic way to write this would be something like:

```python
post_data = {'data': data}

if key is not None:
  post_data['key'] = key
```